react-client-app/src/api/values.js

```diff
@@ -233,13 +235,23 @@ const saveMappingRules = async (scan_report_concept,scan_report_value,table) =>
     else{
         data.source_field = scan_report_value.scan_report_field
     }
-    //_source_concept_id
-    data.omop_field = destination_field.id
-    promises.push(usePost(`/mappingrules/`,data))
+      
     //_concept_id
-    let tempOmopField = await cachedOmopFunction(fields,domain+"_concept_id")
-    data.omop_field = tempOmopField.id
+    data.omop_field = destination_field.id
     promises.push(usePost(`/mappingrules/`,data))
+
+    //_source_concept_id
+    //check if this table is in the list of tables that should be skipped (i.e. have no _source_concept_id)
+    //note - I see no need for keeping a rule for the source_concept_id for any table anymore
+    //     - https://github.com/CO-CONNECT/mapping-pipeline/issues/406
+    //     - Calum 25/02/2022
+    let tempOmopField; 
+    if(! m_skip_source_concept_id.includes(omopTable.table)){
+         tempOmopField = await cachedOmopFunction(fields,domain+"_source_concept_id")
+	 data.omop_field = tempOmopField.id
+	 promises.push(usePost(`/mappingrules/`,data))
+     }
```

Comment:
switch the code to do the source concept here instead of 

side note: we really dont need source concept anyway (need to still triple check with Phil/Tom)